messaging-experiments.yaml

```diff
@@ -200,3 +200,118 @@
                   label: Start Browsing
                   action:
                     navigate: true
+- id: MULTI-STAGE-ABOUTWELCOME-ZERO-STAGE-TEST
+  enabled: true
+  # The filter `!('trailhead.firstrun.didSeeAboutWelcome'|preferenceValue)` is used to exclude
+  # users who have switched channels or re-installed the browser without having a new profile
+  targeting: env.version >= '81.' && env.channel == 'default' && localeLanguageCode == "en" && ((isFirstStartup && !('trailhead.firstrun.didSeeAboutWelcome'|preferenceValue)) || 'zero-stage-test' in activeExperiments)
+  arguments:
+    slug: zero-stage-test
```

Comment:
Should be updated to use slug from experimenter, might be worth filing experimenter ticket now